js-exercises/on-change/onChange.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable quotes */
+const onChange = (obj, onChangeFn) => {
+  const handler = {
+    get(target, property, receiver) {
+      const value = Reflect.get(target, property, receiver);
+      if (typeof value === "object") {
+        return new Proxy(value, handler);
+      }
```

Comment:
This piece of code will throw an error because what if the value is null. type of null is 'object'.
So, the code inside if block get executed and javascript throws an error because the proxy is not created for primitive value.
So. also put check whether the value is null or not along with type checking.